jerry-core/api/jerry.c

```diff
@@ -954,20 +954,19 @@ jerry_create_error_from_value (jerry_value_t value, /**< api value */
      * binary size rather than performance. */
     if (!jerry_value_is_abort (value))
     {
-      return (release ? value : jerry_acquire_value (value));
+      return release ? value : jerry_acquire_value (value);
     }
 
     value = jerry_get_value_from_error (value, release);
     release = true;
```

Comment:
Just a question: is there an existing test for this case?

Reply:
No, there isn't. I will add it, too.